ocs_ci/utility/lvmo_utils.py

```diff
@@ -56,3 +63,68 @@ def lvmo_health_check(tries=20, delay=30):
         delay=delay,
         backoff=1,
     )(lvmo_health_check_base)()
+
+
+def get_sno_disks_by_path():
+    """
+    Get list of storage devices by it's path as listed on node
+
+    Returns:
+        list: list of storage devices full-path (str)
+    """
+    storage_disks_count = config.DEPLOYMENT.get("lvmo_disks")
+    oc_obj = OCP()
+    disks = oc_obj.exec_oc_debug_cmd(
+        node=constants.SNO_NODE_NAME, cmd_list=[LS_DEVICE_BY_PATH]
+    )
+    raw_disks_list = disks.split("\n")
+    raw_disks_list = list(filter(None, raw_disks_list))
+    disks_by_path = list()
+    for line in raw_disks_list[-storage_disks_count:]:
+        if "sda" not in line:
+            disk_name = "/dev/disk/by-path/" + line
+            disks_by_path.append(disk_name)
+
+    return disks_by_path
+
+
+def get_sno_blockdevices():
```

Comment:
I would suggest passing the node name as a argument and use constant as a default value to it.

Reply:
added + docstring